docs/Developers/Executing_Tests.rst

```diff
@@ -34,11 +34,7 @@ You can then execute our tests with
 
 ::
 
-    $ py.test
-
-.. note::
-    If ``py.test`` seems to give errors, try running ``python3 -m pytest``
-    instead.
```

Comment:
@shreyans800755 Hmm... I think like @SanketDG: Let's keep that. But with `py.test` also changed to `pytest` of course ;P